action.yml

```diff
@@ -24,7 +24,7 @@ inputs:
   enable_github_action_token:
     description: "Enable keyless authentication using github action token"
     required: false
-    default: "false"
+    default: "true"
```

Comment:
I've noticed that the README file still uses api_key/password in the example. Perhaps we could replace it with this option instead now?